### PR TITLE
Added full functionality to settings tab & redesigned SignUp/LogIn screen.

### PR DESCRIPTION
**Pull request by MartinCepulionis**
Now you can change password, email, university or status. 

Fixed some older logics issues.

### CODE REVIEW

**Comment by MartinCepulionis** [comment]
In last commit I redesigned SignUp/LogIn screen to a similar UI that is used in MainForm.

File: University-advisor/Forms/MainForm.cs [hunk 1]
```diff
@@ -15,9 +15,12 @@ namespace University_advisor.Forms
     public partial class MainForm : Form
     {
         string currentUser;
+        ArrayList statusList = new ArrayList() { "Student", "Graduate", "Lecturer" };
```
Comment (avizen-j) [inline University-advisor/Forms/MainForm.cs:18]: Use implicitly typed variables, so var statusList.

File: University-advisor/Forms/MainForm.cs [hunk 2]
```diff
@@ -164,34 +175,101 @@ private void ChangeEmail_Click(object sender, EventArgs e)
...
         }
 
         private void ChangeStatus_Click(object sender, EventArgs e)
         {
             errorMessageStatus.Hide();
-            if (!statusBox.SelectedItem.Equals("Current status in DB"))
+
+            string sqlGetCurrentStatus = "SELECT status from users where username='" + currentUser + "';";
+            ArrayList statusFromDB = SqlDriver.Fetch(sqlGetCurrentStatus);
+            string currentStatus = ((Dictionary<string, object>)statusFromDB[0])["status"].ToString();
```
Comment (avizen-j) [inline University-advisor/Forms/MainForm.cs:221]: In this lines use var declaration.

File: University-advisor/Forms/MainForm.cs (hunk 1 above)
Comment (MartinCepulionis) [reply University-advisor/Forms/MainForm.cs:18]: Why is that better? You used ArrayList in SignUpForm

File: University-advisor/Forms/MainForm.cs [hunk 3]
```diff
@@ -15,9 +15,12 @@ namespace University_advisor.Forms
     public partial class MainForm : Form
     {
         string currentUser;
+        ArrayList statusList = new ArrayList() { "Student", "Graduate", "Lecturer" };
+
         public MainForm(string username)
         {
             InitializeComponent();
+            SetValues();
```
Comment (avizen-j) [inline University-advisor/Forms/MainForm.cs:23]: Comment is not related to this line however, you have to change things in this lines:
25) Give it more meaningful name.
44, 79) Delete useless methods.
137) Use implicit declaration.

**Review by avizen-j — CHANGES_REQUESTED** [review]
Some code refactoring needs to be done.

File: University-advisor/Forms/MainForm.cs (hunk 2 above)
Comment (MartinCepulionis) [reply University-advisor/Forms/MainForm.cs:221]: Is var really that better? In this case it is easier to understand what object would be returned. var obscures the actual variable type. If the initializer doesn’t return a clearly defined type then you may not be able to tell the variable’s type.

File: University-advisor/Forms/MainForm.cs (hunk 1 above)
Comment (avizen-j) [reply University-advisor/Forms/MainForm.cs:18]: Then it needs to be changed :)

File: University-advisor/Forms/MainForm.cs (hunk 2 above)
Comment (avizen-j) [reply University-advisor/Forms/MainForm.cs:221]: .NET 3.0 and higher offers an option to use this kind of declaration. Why you afraid to use it? If you are interested in a type you can just place cursor on this variable and you will find out. But why is it better:  "When the compiler encounters fields during code compilation, it records each field's type before processing any expressions associated with it." 
So basically compiler does not have to think about types which is definitely better. More on that in Microsoft documentation: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables

File: University-advisor/Forms/MainForm.cs (hunk 3 above)
Comment (MartinCepulionis) [reply University-advisor/Forms/MainForm.cs:23]: Fixed.

File: University-advisor/Forms/MainForm.cs (hunk 1 above)
Comment (MartinCepulionis) [reply University-advisor/Forms/MainForm.cs:18]: var can only be used when a **local** variable is declared and initialized in the same statement

File: University-advisor/Forms/MainForm.cs (hunk 2 above)
Comment (MartinCepulionis) [reply University-advisor/Forms/MainForm.cs:221]: Thanks for taking your time and explaining the usage of var. I will try to use it more often, but this time I will leave it as it is

**Comment by MartinCepulionis** [comment]
Fixed all the issues from code review.

**Pull request merged**